### PR TITLE
Bump to Android API 25

### PR DESCRIPTION
**Pull request by rogerhu**
Bump Gradle versions

### CODE REVIEW

**Comment by rogerhu** [comment]
Finally passes!!

**Comment by rogerhu** [comment]
ping

File: ParseLiveQuery/build.gradle [hunk 1]
```diff
@@ -8,11 +8,11 @@ version = '1.0.1-SNAPSHOT'
 
 buildscript {
     repositories {
-        mavenCentral()
+        jcenter()
```
Comment (mmimeault) [inline ParseLiveQuery/build.gradle]: Which library wasn't available on mavenCentral()?

Comment (rogerhu) [inline ParseLiveQuery/build.gradle]: jcenter is a superset of maven central.

Android Studio by default has moved to jcenter

https://blog.bintray.com/2015/02/09/android-studio-migration-from-maven-central-to-jcenter/

Comment (mmimeault) [inline ParseLiveQuery/build.gradle]: Yes ok. I personally already experienced connections problems with jcenter over mavenCentral.
But I'm all fine by putting back jcenter.
Thanks

**Pull request merged**